examples/calibration_example.md

```diff
@@ -0,0 +1,111 @@
+## Full example: *post-hoc* Laplace on a large image classifier
+
+An advantage of the Laplace approximation over variational Bayes and Markov Chain Monte Carlo methods is its *post-hoc* nature. That means we can apply LA on (almost) any *pre-trained* neural network. In this example, we will see how we can apply the last-layer LA on a deep WideResNet model, trained on CIFAR-10.
+
+#### Data loading
+
+First, let us load the CIFAR-10 dataset. The helper scripts for CIFAR-10 and WideResNet are available in the `examples/helper` directory in the main repository.
+
+```python
+import torch
+import numpy as np
+import helper.wideresnet as wrn
+import helper.dataloaders as dl
+from netcal.metrics import ECE
+import urllib.request
+import os.path
+
+from laplace import Laplace
+
+
+np.random.seed(7777)
+torch.manual_seed(7777)
+torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.benchmark = True
+
+train_loader = dl.CIFAR10(train=True)
+test_loader = dl.CIFAR10(train=False)
+targets = torch.cat([y for x, y in test_loader], dim=0).numpy()
+```
+
+#### Load a pre-trained model
+
+Next, we will load a pre-trained WideResNet-16-4 model. Note that a GPU with CUDA support is needed for this example.
+
+``` python
+model = wrn.WideResNet(16, 4, num_classes=10).cuda().eval()
+
+# Download pre-trained model if necessary
+if not os.path.isfile('CIFAR10_plain.pt'):
+    if not os.path.exists('./temp'):
+        os.makedirs('./temp')
+
+    urllib.request.urlretrieve('https://nc.mlcloud.uni-tuebingen.de/index.php/s/2PBDYDsiotN76mq/download', './temp/CIFAR10_plain.pt')
```

Comment:
Maybe these 4 lines can be hidden away in the `helper` methods and wrapped in `download_model_temp()` or something alike?